src/test/scala/cognite/spark/v1/AssetsRelationTest.scala

```diff
@@ -132,7 +132,9 @@ class AssetsRelationTest extends FlatSpec with Matchers with SparkTest {
         |'$assetsTestSource' as source,
         |10 as id,
         |0 as createdTime,
-        |0 as lastUpdatedTime
+        |0 as lastUpdatedTime,
```

Comment:
Shouldn't `null` be allowed for this, as well as for `rootId`?

Reply:
It should be, but it isn't now because it's using the sdk case class as schema (updateTimes aren't Options).

Reply:
Can we fix that? Inserts should be based on the `Create` classes, fields that are in the read class but not in the create class should be made optional. Maybe a separate PR though.

Reply:
Absolutely. I agree, let's do it in a different PR, and do it for all the resource types.

Reply:
Emil was helping me use the `Create` classes for this earlier, still working on it then will make pr